src/sentry/feedback/usecases/create_feedback.py

```diff
@@ -341,6 +341,16 @@ def create_feedback_issue(
             sample_rate=1.0,
         )
 
+    # Removes associated_event_id from event if it is invalid
+    associated_event_id = get_path(event, "contexts", "feedback", "associated_event_id")
+
+    if associated_event_id:
+        try:
+            UUID(str(associated_event_id))
+        except ValueError:
+            associated_event_id = None
+            event["contexts"]["feedback"]["associated_event_id"] = None
```

Comment:
```suggestion
            event["contexts"]["feedback"].pop("associated_event_id", "")
```
Nit - I'm not sure, but there might be downstream validation that the id, if it exists, is a string